backend/infrahub/core/schema/definitions/core.py

```diff
@@ -942,6 +942,10 @@
                     "enum": PermissionDecision.available_types(),
                     "default_value": PermissionDecision.ALLOW_ALL.value,
                     "order_weight": 5000,
+                    "description": (
+                        "Decide whether to deny or allow the action. If allowed, it can be configured for the default branch, any other "
```

Comment:
will it show on both global and object permissions?

for the global permissions, we only have "allow" and "deny" so the part for the branches is not relevant, right?

Reply:
Good point